docs/api/search.md

```diff
@@ -0,0 +1,107 @@
+Search
+======
+
+####Our api supports searching artifact metadata which will return a link header for each search result.
+
+####Note:
+    * The only requirement is the search portion of the criteria.
+
+####Search criteria:
+    * We currently support equality, wildcard, and version searches.
+    * Equality search syntax: `"field=value"`
+    * Wildcard search syntax: `"field=valu*"` where the `*` represents the 0 or more characters.
+    * Version search syntax: `"field~=1.1"` where search results >= 1.1 but < 2.
+    * We support escaping of \*, ~, and = for literal evaluation.
+
+####Sort criteria:
+    * We support sort types and sort flags.
+    * Sort types supported are ASC and DESC
+        * with aliases ASCENDING and DESCENDING respectively
+    * Sort flag supported is VERSION (with alias VER).
+    * This version flag uses distutils.version.LooseVersion library (where 1.19 > 1.2).
+        * More info: https://docs.python.org/2/distutils/apiref.html#module-distutils.version
+    * The sort criteria must start with the field name.
+    * The default sort type is ascending. If multiple sort types are given the last is used.
+    * With multi-sorts the first sort takes precedence.
+
+####Limit:
+    * A limit can be set on the number of results returned back by the api.
+    * Note: the method of sort effects which artifacts are contained within the limit.
+
+####Artifact Path:
+    * This api exposes two search endpoints.
+        * Search from the root of an S3 bucket:  **/<bucket-name>/artifact/\_search**
+        * Search from a partial path: **/<bucket-name>/artifact/<path>/\_search**
+
+#### Example of criteria with single search and sort with a limit on number of results.
+
+    {
```

Comment:
I see you HAVE added these below.  You can either just remove this section, or put it last.  If it would be me, I would be looking for examples and maybe come back for advanced usage of criteria